web/scores_test.go

```diff
@@ -5,36 +5,75 @@ package web
 
 import (
 	"encoding/json"
+	"net/http/httptest"
+	"testing"
+
 	"github.com/Team254/cheesy-arena-lite/field"
 	"github.com/Team254/cheesy-arena-lite/game"
 	"github.com/stretchr/testify/assert"
-	"net/http/httptest"
-	"testing"
 )
 
 func TestGetScores(t *testing.T) {
 	web := setupTestWeb(t)
 
 	score1 := game.TestScore1()
 	score2 := game.TestScore2()
-	web.arena.RedScore.AutoPoints = score1.AutoPoints
-	web.arena.RedScore.TeleopPoints = score1.TeleopPoints
-	web.arena.RedScore.EndgamePoints = score1.EndgamePoints
-	web.arena.BlueScore.AutoPoints = score2.AutoPoints
-	web.arena.BlueScore.TeleopPoints = score2.TeleopPoints
-	web.arena.BlueScore.EndgamePoints = score2.EndgamePoints
+
+	web.arena.RedScore.Taxi[0] = game.AutonTaxiValues[score1.Taxi[0]]
+	web.arena.RedScore.Taxi[1] = game.AutonTaxiValues[score1.Taxi[1]]
```

Comment:
`AutonTaxiValues[...]` seems unnecessary - I think `score1.Taxi[n]` is already a `AutonTaxiStatus`?
```suggestion
	web.arena.RedScore.Taxi[0] = score1.Taxi[0]
	web.arena.RedScore.Taxi[1] = score1.Taxi[1]
```